src/iocore/hostdb/HostDB.cc

```diff
@@ -784,9 +826,9 @@ HostDBContinuation::lookup_done(TextView query_name, ts_seconds answer_ttl, SRVH
 {
   ink_assert(record);
   if (query_name.empty()) {
-    if (is_byname()) {
+    if (hash.is_byname()) {
       Dbg(dbg_ctl_hostdb, "lookup_done() failed for '%.*s'", int(hash.host_name.size()), hash.host_name.data());
-    } else if (is_srv()) {
+    } else if (hash.is_srv()) {
```

Comment:
Can you explain why these various functions are now all called via `hash`?

Reply:
I moved these utility functions to `HostDBHash` so I could use them without a `HostDBContinuation`

Reply:
Oh, I see. I saw in your patch that functions got moved, but I didn't notice it was these and that they were moved to a new class. Thank you for the explanation.